WordPress/Classes/ViewRelated/Me/All Domains/View Models/AllDomainsListViewModel.swift

```diff
@@ -107,8 +119,8 @@ class AllDomainsListViewModel {
         return .init(title: title, description: description, button: button)
     }
 
-    /// The empty state to display when there are no domains matching the search query.
-    private func emptyStateViewModel(searchQuery: String) -> AllDomainsListEmptyStateViewModel {
+    /// The message to display when there are no domains matching the search query.
+    private func emptyMessageViewModel(searchQuery: String) -> AllDomainsListMessageStateViewModel {
```

Comment:
[Nit]
```suggestion
    private func noSearchResultsMessageViewModel(searchQuery: String) -> AllDomainsListMessageStateViewModel {
```

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/21910/commits/2e019c7c962e66fe87c05f8d36048b273a169754